Mifare Classic Tool/app/build.gradle

```diff
@@ -4,7 +4,6 @@ plugins {
 
 android {
     compileSdkVersion 31
-    buildToolsVersion '31.0.0'
```

Comment:
Is safe to remove this? What build tools version will be used when it's not declared?

Reply:
Latest.